docs/user/reference/features.rst

```diff
@@ -1,7 +1,56 @@
-========
-Features
-========
+=================
+Feature reference
+=================
 
+.. TODO: Continue to add more features here.
+
+Hosting features
+----------------
+
+⏩️ :doc:`/custom-domains`
+  Documentation projects can use their own domain name.
+  A project may define multiple domains,
+  as well as define the *canonical domain* where all other domains will redirect.
+
+
+⏩️ :doc:`/versions`
+  We support multiple versions and translations,
+  integrated nicely into the URL of your documentation.
+  This is served at ``/en/latest/`` by default.
+  If you only have 1 version and translation,
+  we also support :doc:`single version projects </single_version>` served at ``/``.
+
+
+⏩️ :doc:`/user-defined-redirects`
+  Projects may define their own custom URL redirects,
+  with advanced functionality like folder redirects.
+
+⬇️ :doc:`/reference/cdn`
```

Comment:
```suggestion
⏩️ :doc:`/reference/cdn`
```

Before, I used the arrow-down emoji to denote that the content was in an anchor (further down the page).

I think it makes sense to keep the emojis as "⏩️" for now. We could potentially invest some time in finding other emojis... but I'd say that it's a huge distraction or now.

However, the visual touch of the emoji - IMO - makes this list lighter to read.

Reply:
Yea, I just copied these and didn't notice the emoji difference in my editor :)